AGENTS.md

```diff
@@ -2,71 +2,72 @@
 
 ## Migrating to the new authoring layer and the dbt Fusion engine
 
-if a user says "migrate me to the new authoring layer" what that really means is that they want to have the following commands complete with no errors (i.e. exit code 1).
+if a user says "migrate my dbt project to the new authoring layer" or "make my dbt project compatible with the Fusion engine" do the following steps and the migration is only successfully completed once `dbtf compile` finishes with 0 errors. Create a changes_made.md file documenting and summarizing all of the code changes made in the user's dbt project, including what the error was, the fix applied, and a summary of why the fix was chosen.
 
-1. `dbtf debug`
-2. `dbtf parse --show-all-deprecations`
-3. `dbtf compile --static-analysis off`
-4. `dbtf compile`
-6. `dbtf test`
+Before you begin, please read and understand the resources section below. You should strictly use the resources below to update the project.
 
-the best way to do this today is to start with a migration wizard that's build into the dbt Fusion CLI.
-```
-dbtf init --fusion-upgrade
-```
+1. Run `dbtf debug` in the terminal to check their data platform connections. Proceed to step 2 if there are no errors. If there are errors, please summarize the error succinctly so the user knows how to debug on their own.
+2. Run `dbtf parse --show-all-deprecations` in the terminal to check for compatibility errors in their current project. Summarize the log output by specifying how many errors were found and group the errors in a way that's easily understandable.
+3. Install [dbt-autofix](https://github.com/dbt-labs/dbt-autofix) and run autofix to try to fix errors. Prefer uv/uvx to install (`uv tool install dbt-autofix`) and run the package (`uvx dbt-autofix deprecations`), but fall back to pip and other methods if needed. Summarize the results of the autofix run and include how many errors were resolved. Run `dbtf parse` again to check for remaining errors and summarize with how many errors were found and a brief summary of the types of errors.
+4. For remaining errors, please ONLY use the resources below to attempt to resolve them. If you can't figure out a fix from the resources below, notify the user and break out of the flow. Attempt the fixes error by error, grouping similar errors based on the error code and message. You should also summarize which error you're working on in the chat to give users context. 
 
-This command will first make sure you have a `profiles.yml`, before attempting the above commands in order.
+   **Special handling for common unsupported features:**
+   - **Iceberg table errors**: These are not config errors but references to unsupported table types. See iceberg_tables.md for guidance.
+   - **Python model errors**: Disable with `{{ config(enabled=false) }}` at the top of the file.
+   - **Custom materialization errors**: Disable with `{{ config(enabled=false) }}` at the top of the file.
```

Comment:
custom materializations are supported now! to get the project working, they'll likely have to follow the [custom materialization guidance](https://github.com/dbt-labs/dbt-autofix/blob/main/manual_fixes/custom_configuration.md)

if you've come across unmigratable custom materializations, then we should open tickets!